### PR TITLE
Adding "emailuser" to TeamworkUserIdentity

### PR DESCRIPTION
**Pull request by sumanac**
Adding "emailuser" to TeamworkUserIdentity

### CODE REVIEW

File: api-reference/beta/resources/teamworkuseridentity.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ Inherits from [identity](../resources/identity.md).
 |:---|:---|:---|
 |displayName|String|Inherited from [identity](../resources/identity.md). Display name of the user. Optional.|
 |id|String|Inherited from [identity](../resources/identity.md). ID of the user. |
-|userIdentityType|teamworkUserIdentityType| Type of user. Possible values are: `aadUser`, `onPremiseAadUser`, `anonymousGuest`, `federatedUser`, `personalMicrosoftAccountUser`, `skypeUser`, and `phoneUser`.|
+|userIdentityType|teamworkUserIdentityType| Type of user. Possible values are: `aadUser`, `onPremiseAadUser`, `anonymousGuest`, `federatedUser`, `personalMicrosoftAccountUser`, `skypeUser`, `phoneUser`, `unknownFutureValue` and `emailUser`.|
```
Comment (sumanac) [inline api-reference/beta/resources/teamworkuseridentity.md:26]: Was this property missing from beta for any reasons? I added it. I can remove it if this was intentional.

Comment (AkJo) [reply api-reference/beta/resources/teamworkuseridentity.md:26]: Are we not describing the meaning of these userIdentityTypes somewhere?

**Review by AkJo — APPROVED** [review]

File: api-reference/beta/resources/teamworkuseridentity.md (hunk 1 above)
Comment (RamjotSingh) [reply api-reference/beta/resources/teamworkuseridentity.md:26]: `unknownFutureValue` was removed because beta never get it.

**Review by RamjotSingh — CHANGES_REQUESTED** [review]

File: api-reference/beta/resources/teamworkuseridentity.md (hunk 1 above)
Comment (sumanac) [reply api-reference/beta/resources/teamworkuseridentity.md]: Removed from beta.

**Review by RamjotSingh — APPROVED** [review]

**Review by Lauragra — APPROVED** [review]

**Pull request merged**